package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "babel-runtime": "^5.6.20",
+    "graphql-relay": "^0.3.2",
```

Comment:
Something like that possible yeah, i've just been happy with being able to not depend on `relay` directly and let the user define the version he needs to a certain degree.

I'd love this lib to be a helper when needed than something you have to use and customize for all fields (not sure if that would be an issue with the graphql-relay package actually).

A reason to not use own graphql package is also that stuff like `instanceof GraphQLList` would fail since the constructor would differ between two packages - Might not be an issue with relay.

Lets give having it as a straight dependency a go and possible test if it works when the user has his own version installed aswell.